packages/Meshfree/src/DTK_DetailsNearestNeighborOperatorImpl.hpp

```diff
@@ -85,15 +85,14 @@ struct NearestNeighborOperatorImpl
 
         buffer_indices = import_target_indices;
         buffer_ranks = import_ranks;
-        Kokkos::realloc( buffer_values, n_imports,
-                         source_values.dimension_1() );
+        Kokkos::realloc( buffer_values, n_imports, source_values.extent( 1 ) );
         Kokkos::parallel_for(
             DTK_MARK_REGION( "get_source_values" ),
             Kokkos::RangePolicy<ExecutionSpace>( 0, n_imports ),
             KOKKOS_LAMBDA( int i ) {
-                for ( int j = 0; j < (int)source_values.dimension_1(); ++j )
-                    buffer_values( i, j ) =
-                        source_values( import_source_indices( i ), j );
+                for ( int j = 0; j < (int)source_values.extent( 1 ); ++j )
+                    buffer_values.access( i, j ) =
+                        source_values.access( import_source_indices( i ), j );
```

Comment:
Comment that using `View::access()` is just a workaround and that we could/should write two specializations of the kernel.